.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,30 @@
+name: Bug Report
+description: File a bug report
```

Comment:
what is a "bug report" in the context of the CIP repo?  Do we even want this template?

Reply:
Automations being broken.